### ISSUE TITLE
Pre-trained model shows more CER than when it was trained and saved

### ISSUE THREAD

**Issue by bishwa420** [SeanNaren/deepspeech.pytorch#173]
This may sound strange, but, the model had been training for about a month (of course there is no GPU) and then saved with the best validation performance:

> Model name:          run_1_on_main.pth.tar
> DeepSpeech version:  0.0.1
> 
> Recurrent Neural Network Properties
>   RNN Type:          gru
>   RNN Layers:        7
>   RNN Size:          768
>   Classes:           29
> 
> Model Features
>   Labels:             _CGOSTUXabcdefghijklnoprstuy
>   Sample Rate:       16000
>   Window Type:       hamming
>   Window Size:       0.02
>   Window Stride:     0.01
> 
> Training Information
>   Epochs:            47
>   Current Loss:      0.032
>   Current CER:       3.693
>   Current WER:       42.079
> 

But when I loaded the model (25+- days after being saved) from the disk, and ran the **test.py** with the same validation data, it gave me CER more than 15%. I was flabbergasted. What could be the possible reason?

PS: My machine is Ubuntu 16.04.

**Comment by ryanleary** [comment]
I see you closed the issue. Did you figure it out?

**Comment by bishwa420** [comment]
Yes, it was my bad that I didn't divide the CER by the number of test examples. I wanted to delete this issue, but that's what I cannot. Thatswhy I closed this issue.

**Comment by ryanleary** [comment]
Perfect! Glad you figured it out.

Issues generally can't be deleted, so it's good practice if reporting an issue and coming to a resolution on it to indicate what the solution was. Cheers!
